isso/js/tests/unit/template.test.js

```diff
@@ -0,0 +1,15 @@
+// TODO not working because jest and pug-loader somehow don't want to work
```

Comment:
Do you have the same error as me?
```
$ npm run test-unit

> test-unit
> jest --config isso/js/jest.config.js isso/js/tests/unit/

 PASS  isso/js/tests/unit/utils.test.js
 PASS  isso/js/tests/unit/config.test.js
 FAIL  isso/js/tests/unit/template.test.js
  ● Template functions

    ReferenceError: nodeResolve is not defined

      at Object.fn [as postbox] (eval at <anonymous> (app/template.js:28:13), <anonymous>:6:21)
      at Object.render (app/template.js:81:10)
      at Object.<anonymous> (tests/unit/template.test.js:13:19)
```

Reply:
Yeah `pug-loader` and Jest will not work together out of the box. This test is currently expected to fail.

Reply:
Disabled this test for onw